packages/roosterjs-editor-api/lib/table/applyCellShading.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeSource, IEditor, ModeIndependentColor } from 'roosterjs-editor-types';
+import { VTable } from 'roosterjs-editor-dom';
+
+/**
+ * Apply cell shading on a vTable selection
```

Comment:
Is this function for cell or the whole table?

Reply:
Only cell and if there is a selection of cells